app/mailers/feedback_mailer.rb

```diff
@@ -7,6 +7,7 @@ def feedback(sending_user, form_values)
       to: self.class.support_email,
       subject: 'Feedback submission',
       from: from,
+      cc: from,
```

Comment:
Yeah, I'm not sure where that from header is being overwritten so that it doesn't send from the user. I didn't want to change the from value in the bugfix because it wasn't necessary to actually resolve the issue, but we can flag it as a # FIXME to change that value so that it reflects the actual behavior of the application?